Protacon.RxMq.AzureServiceBus/Topic/AzureTopicSubscriber.cs

```diff
@@ -45,14 +45,15 @@ private class Binding<T> : IDisposable, IBinding where T : new()
                     {
                         try
                         {
+                            long arrival = DateTimeOffset.UtcNow.ToUnixTimeMilliseconds();
                             var body = Encoding.UTF8.GetString(message.Body);
 
                             if (!_excludeTopicsFromLogging.Contains(topicName))
                             {
-                                logging.LogInformation("Received '{subscription}': {body} with Azure MessageId: '{messageId}'", subscriptionName, body, message.MessageId);
+                                logging.LogInformation("Received '{subscription}': {body} with Azure MessageId: '{messageId}'. Arrival time in UnixTimeMilliseconds: '{arrival}'", subscriptionName, body, message.MessageId, arrival);
```

Comment:
Aika verbose, itse suosisin
```suggestion
                                logging.LogInformation("Received '{subscription}': {body} with Azure MessageId: '{messageId}'. Arrival: '{arrival}'", subscriptionName, body, message.MessageId, arrival);
```

Reply:
Verbosea ehkä mutta jos sovelluksen toimintaa pitää yrittää tulkita pelkästään lokitekstin perusteella niin pelkkä "Arrival" on aika heikko tietue, jonka perusteella tehdä tulkintoja. Kaikki ei osaa/kykene/halua lähteä sorsaa parsimaan ja etsimään että mikä hemmetin numero on kyseessä. Periaatteessa numeron koko kertonee jo aika paljon mutta se olis silti arvaus kun tämä lokirivi kertoo sen eksplisiittisesti.